docs/ai-gateway/analytics/moesif-analytics.md

```
@@ -0,0 +1,126 @@
+| `als_server_port`  | int     | Yes      | - | gRPC port on which the ALS server listens for incoming access log streams from Envoy.      |
+| `shutdown_timeout` | int     | No       | `600` | Maximum time allowed for the ALS server to gracefully shut down while completing in-flight log processing(in seconds). |
+| `als_plain_text`   | boolean | No       | `true` | Use plaintext gRPC        |
+| `public_key_path`   | string | No       | - | Path to the public key used for securing ALS communication when transport-level encryption or authentication is enabled.        |
+| `private_key_path`   | string | No       | - | Path to the privare key used for securing ALS communication when transport-level encryption or authentication is enabled.        |
+| `max_message_size` | int     | No       | `1000000000` |Maximum size of a single gRPC message that the ALS server is allowed to receive from Envoy.      |
+| `max_header_limit` | int     | No       | `8192` | Maximum allowed size of request or response headers processed by the ALS server      |
+
+
+## Configuration Examples
+
+#### Integrate Moesif Publisher
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)